frontend/Dockerfile

```diff
@@ -22,7 +22,9 @@ WORKDIR /home/node/
 
 # Copy monorepo mocking files into `/home/node`, which pretends to be the monorepo root.
 # Note: these files must be manually un-ignored in the root .dockerignore
+# hadolint ignore=DL3022
 COPY --from=repo_root --chown=node:node .npmrc .pnpmfile.cjs pnpm-lock.yaml tsconfig.base.json ./
```

Comment:
Where is `repo_root` coming from here?

Reply:
This is passed as an additional build context to the `build-img` recipe...

https://github.com/WordPress/openverse/blob/722006eb981999743d4950c1c4bf5bafc945f473/frontend/justfile#L36-L43

...and also set in the CI + CD job when the frontend image is supposed to build.

https://github.com/WordPress/openverse/blob/722006eb981999743d4950c1c4bf5bafc945f473/automations/python/workflows/set_matrix_images.py#L76-L80